mirage/fixtures/external-storage-services.ts

```diff
@@ -0,0 +1,124 @@
+// TODO: needs iconUri and authUri
+export default [
+    {
+        id: 'bitbucket',
+        name: 'Bitbucket',
+        readOnly: true,
+        supportsCopy: false,
+        supportsUserSpecificRootFolder: false,
+        supportsFileVersioning: false,
+        supportsBulkDownload: false,
+        maxConcurrentDownloads: 2,
+        maxUploadMb: 0,
+    },
+    {
+        id: 'box',
+        name: 'Box',
+        readOnly: false,
+        supportsCopy: true,
+        supportsUserSpecificRootFolder: false,
+        supportsFileVersioning: false,
+        supportsBulkDownload: false,
+        maxConcurrentDownloads: 2,
```

Comment:
This may only be 1. Did you get this from the file code?

Reply:
I just used the `parallelUploadsLimit` in [provider-file.ts](https://github.com/futa-ikeda/ember-osf-web/blob/cf7035986f68fb0b09ad7d50702e7b4f4d76776f/app/packages/files/provider-file.ts#L20) for these values. I will update this to 1